test/bin/build_images.sh

```diff
@@ -712,12 +725,19 @@ PREVIOUS_RELEASE_VERSION=$(echo "${previous_version_repo}" | cut -d, -f1)
 PREVIOUS_RELEASE_REPO=$(echo "${previous_version_repo}" | cut -d, -f2)
 
 RHOCP_MINOR_Y=""
-RHOCP_MINOR_Y1=""
+RHOCP_MINOR_Y_BETA=""
 if is_rhocp_available "${MINOR_VERSION}"; then
     RHOCP_MINOR_Y="${MINOR_VERSION}"
+elif is_rhocp_beta_available "${MINOR_VERSION}"; then
+    RHOCP_MINOR_Y_BETA="https://mirror.openshift.com/pub/openshift-v4/${UNAME_M}/dependencies/rpms/4.${MINOR_VERSION}-el9-beta/"
```

Comment:
Can it be that none of these are available and we should exit with error?

Reply:
If we don't enable RHOCP with good CRI-O, then blueprints won't be depsolved successfully. Hard to say if we should exit here - I'd say no, because we want to give a chance to older RHOCPs, like 4.17 for arm which now is not available